packages/injective-math/src/utils.rs

```diff
@@ -65,3 +66,33 @@ pub fn ensure_band<T: Ord + Display>(v: &T, min: Option<&T>, max: Option<&T>, ra
 pub fn band_error_to_human(err: StdError, value_name: &str) -> StdError {
     StdError::generic_err(format!("Value '{value_name}' failed validation due to: '{err}'"))
 }
+
+pub fn div_dec(num: FPDecimal, denom: FPDecimal) -> FPDecimal {
```

Comment:
please add some tests for the three new functions, they can be one simple test each, nothing fancy